core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -39,20 +41,13 @@ import org.apache.spark.util.Utils
  * has started will events be actually propagated to all attached listeners. This listener bus
  * is stopped when `stop()` is called, and it will drop further events after stopping.
  */
-private[spark] class LiveListenerBus(conf: SparkConf) extends SparkListenerBus {
-
-  self =>
+private[spark] class LiveListenerBus(conf: SparkConf) {
 
   import LiveListenerBus._
 
   private var sparkContext: SparkContext = _
```

Comment:
maybe I missed it but I don't see this being set anywhere now

Reply:
Ah, this should be set in `start()`, probably deleted in one of the code moves.